cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-item-status.cy.js

```diff
@@ -0,0 +1,56 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
+    const testData = { csvFileName: 'cypress/fixtures/item_barcodes9.csv' };
```

Comment:
csvFileName property is not used anywhere in the test, it can be either removed or modified to be used in 'BulkEditSearchPane.uploadFile('item_barcodes9.csv');' action

Reply:
fixed